spec/imsc-hrm.html

```diff
@@ -472,7 +475,7 @@ <h2>General</h2>
         <li>paint the text or image subtitle content.</li>
       </ol>
 
-      <p>The Presentation Compositor SHALL start rendering E<sub>n</sub>:</p>
+      <p>The Presentation Compositor starts rendering E<sub>n</sub>:</p>
```

Comment:
temporal language consistency nit across TTWG specs: I prefer (because I think it reduces the potential for misunderstanding) to use "begin" for time-related initiation, and "start" for spatial-related origin.